tutorials/kotlin-springboot-app-engine-java8.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Run a Kotlin Spring Boot app on Google App Engine Standard
+description: Learn how to deploy a Kotlin Spring Boot app to Google App Engine Standard.
+author: bshaffer
+tags: App Engine, Kotlin, Spring Boot, Java, App Engine Standard
+date_published: 2018-01-17
+---
+
+[Google App Engine Standard](https://cloud.google.com/appengine/docs/standard/)
+is an easy way to deploy your apps to the same infrastructure that powers
+Google's products. In this tutorial you'll see how to deploy your
+[Kotlin](https://kotlinlang.org/) and
+[Spring Boot](https://projects.spring.io/spring-boot/) application to App Engine
+Standard.
+
+You will create a new Spring Boot application, and then you will learn how to:
+
+*   Deploy your application
+*   Update your application
+
+While the tutorial uses Kotlin 1.2 and Spring Boot 2 M7, other releases of Kotlin and Spring Boot should work
+without any modifications (other than version numbers in Maven files). This tutorial does assume you're familiar
+with Spring Boot and creating web applications. For simplicity the tutorial responds with JSON to a specific HTTP request, but can
+be built-on to connect to other Google services and/or databases.
+
+## Before you begin
+
+Before running this tutorial, you must set up a Google Cloud Platform project,
+and you need to have the Google Cloud SDK installed.
+
+Create a project that will host your Spring Boot application. You can also reuse
+an existing project.
+
+1.  Use the [Google Cloud Platform Console](https://console.cloud.google.com/)
+    to create a new Cloud Platform project. Remember the project ID; you will
+    need it later. Later commands in this tutorial will use `[PROJECT_ID]` as
+    a substitution, so you might consider setting the `PROJECT_ID` environment
+    variable in your shell.
+
+2.  Enable billing for your project.
+
+3.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/). Make sure
+    you [initialize](https://cloud.google.com/sdk/docs/initializing) the SDK
+    and set the default project to the new project you created.
+
+    Version 175.0.0 or later of the SDK is required.
+
+3.  Install [JDK 8 or higher](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) if you do not already have it.
+
+4.  Install [Maven](https://maven.apache.org/install.html).
+
+## Creating a new app and running it locally
+
+In this section, you will create a new Spring Boot app and make sure it runs. If
+you already have an app to deploy, you can use it instead.
+
+1. Use [start.spring.io](https://start.spring.io) to generate a Spring Boot
+application.
+
+    * Select **Kotlin** as the language and **Maven** as the build system.
+    * Using the Dependency selector, type **GCP Support** and select it.
+    * In the same Dependency selector, type **Web** and select first result.
```

Comment:
"... and select the first result..."

Reply:
AS is this.

Reply:
Yeah I didn't make this change not sure how it got in the PR.